Formula/m/mariadb.rb

```diff
@@ -1,10 +1,9 @@
 class Mariadb < Formula
   desc "Drop-in replacement for MySQL"
   homepage "https://mariadb.org/"
-  url "https://downloads.mariadb.com/MariaDB/mariadb-11.0.2/source/mariadb-11.0.2.tar.gz"
-  sha256 "3c71576cad0ea416882270e383f94cc89693b7fbcce1fa4f306fe3e931e464ae"
+  url "https://downloads.mariadb.com/MariaDB/mariadb-11.0.3/source/mariadb-11.0.3.tar.gz"
```

Comment:
downloads.mariadb.com files are now found at dlm.mariadb.com and the new URL for this file is https://dlm.mariadb.com/3399259/MariaDB/mariadb-11.0.3/source/mariadb-11.0.3.tar.gz.

That said, 11.1.2 appears to be the newest stable (GA) version as of 2023-08-21.